### PR TITLE
Ex 03

### PR DESCRIPTION
**Pull request by datapawn**
Finalizes ex-03 without revisions.
Finished verifyphone()
Built and finished verifyemail():  note - verifyemail() does not yet check for more than one @ symbol.  That will need to be fixed in future revisions.
Briskette gave me permission to say it was 'done' though.

### CODE REVIEW

File: Exercises/ex-03/base/remain.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    if len(outnum) > 10:
+        outnum = "Error:  Too many numbers."
+    else:
+        outnum = "({}) {}-{}".format(outnum[:3],outnum[3:6],outnum[6:])
+
+    return outnum
+
+def verifyemail(e):
+    #slice off the last four letters
+    dotcom = e[-4::1]
```
Comment (JayLoomis) [inline Exercises/ex-03/base/remain.py:43]: What if their top level domain isn't a three letter one?

Can you think of a less brittle way to get it?

File: Exercises/ex-03/base/remain.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    
+    #list of Top Level Domains that don't suck
+    tld_list = (".com",".org",".net",".int",".edu",".gov",".mil")
+
+    #check for @ and .something
+    if "@" not in e:
```
Comment (JayLoomis) [inline Exercises/ex-03/base/remain.py]: It would be just as easy to count the number of @s and check for 1.

**Review by JayLoomis — CHANGES_REQUESTED** [review]
A couple of things to look at.

File: Exercises/ex-03/base/remain.py (hunk 1 above)
Comment (datapawn) [reply Exercises/ex-03/base/remain.py:43]: Should i change this to accept TLDs other than the standard list provided?  If there is a specific list I can add them, if there is not, i can change the process by which i slice the end of the data off.

File: Exercises/ex-03/base/remain.py (hunk 2 above)
Comment (datapawn) [reply Exercises/ex-03/base/remain.py]: So i did that. get off my back.

**Review by datapawn — COMMENTED** [review]
Addressed Review Comments Take A Look It's In a Book.

**Comment by JayLoomis** [comment]
There are things to do before this is done done, but it's fine for now.

LGTM

**Pull request merged**